ztp/extra-manifests-builder/04-accelerated-container-startup/test.sh

```diff
@@ -138,4 +138,22 @@ test_unrestrictedCpuset 0 "0-9" ' '                          "0-9" "Empty file f
 test_unrestrictedCpuset 0 "0-9" ''                           "0-9" "Missing file falback"
 test_unrestrictedCpuset 1 ''    ''                           ''    "Fallback file missing"
 
+test_getCPUCount() {
+  local expectedRc="$1" expectedValue="$2" cpuSet="$3"
+  echo "Testing getCPUCount: $4"
+
+  local result
+  result=$(getCPUCount $cpuSet)
+  local rc=$?
+  [[ $rc -eq $expectedRc ]] || fatal "  getCPUCount failed: Expected rc $expectedRC != $rc"
+  [[ $result == $expectedValue ]] || fatal "  getCPUCount failed: Expected return value '$expectedValue' != '$result'"
+}
+
+test_getCPUCount 0 "4" "3-4,54,55"     "Valid CPU List with no ranges"
```

Comment:
But this has a range, `3-4`?

Reply:
Will fix, should be a ',' instead of '-'